lib/spack/spack/cmd/env.py

```diff
@@ -391,6 +395,77 @@ def env_loads(args):
     print('   source %s' % loads_file)
 
 
+def env_update_setup_parser(subparser):
+    subparser.add_argument(
+        metavar='env', dest='env',
+        help='name or directory of the environment to activate'
+    )
+    spack.cmd.common.arguments.add_common_arguments(subparser, ['yes_to_all'])
+
+
+def env_update(args):
+    manifest_file = ev.manifest_file(args.env)
+    backup_file = manifest_file + ".bkp"
+    needs_update = ev.is_latest_format(manifest_file)
```

Comment:
I assume this should be `needs_update = not ev.is_latest_format(manifest_file)` (i.e. add `not`)

Reply:
Thanks for spotting this. Fixed now.